raiden/transfer/channel.py

```diff
@@ -625,7 +631,16 @@ def compute_merkletree_without(merkletree, lockhash):
 
 
 def create_senddirecttransfer(channel_state, amount, identifier):
-    our_balance_proof = channel_state.our_state.balance_proof
+    our_state = channel_state.our_state
+    partner_state = channel_state.partner_state
+
+    msg = 'caller must make sure there is enough balance'
```

Comment:
These assertions here are for a programming error, right? Caller here refers to the function's caller I suppose?

Reply:
Yes, it would be a programming error to call `create_senddirecttransfer` if the channel doesn't have enough capacity.